styles.scss

```diff
@@ -0,0 +1,251 @@
+$backgroundColor: #f5f5f5;
+$darkBackgroundColor: #949494;
+$buttonColor: #5E7FC1;
+$marginAuto: 0 auto;
+$mainPadding: 30px;
+$basicFont: 1rem;
+$biggerFont: 1.3rem;
+$titleFont: 3.5rem;
+$thickBorder: 2px solid;
+$thinBorder: 1px solid;
+
+@mixin button ($backgroundColor: #5E7FC1, $color: #FFFFFF, $border: 2px solid $buttonColor, $shadow: 0 8px 16px 0 rgba(0,0,0,0.2), $transition: 0.4s, $alignment: center, $padding: 15px 32px, 
+                $display: inline-block, $cursor:pointer, $font: 'Montserrat', $fontSize: 16px) {
+    background-color: $backgroundColor;
+    color: $color;
+    border: $border;
+    box-shadow: $shadow;
+    transition-duration: $transition;
+    text-align: $alignment;
+    padding: $padding;
+    cursor: $cursor;
+    font-family: $font;
+    font-size: $fontSize;
+}
+
+@mixin grid ($columns, $rows) {
+    grid-template-columns: $columns;
+    grid-template-rows: $rows;
+}
+
+body {
+    margin: $marginAuto;
+    font: {
+        size: $basicFont;
+        family: 'Montserrat', serif;
+    }
+    line-height: 1.6;
+    width: 100%;
+}
+
+.top-menu {
+    background-color:black;
+    border-bottom: $thickBorder $backgroundColor;
+    justify-content: space-around;
+    display: flex;
+    &__options {
+        display: block;
+        text-align: center;
+        &--home, &--others {
+            list-style: none;
+            padding: 0px 10px;
+            a {
+                text-decoration: none;
+                color: #FFFFFF;    
+            }
+        }
+        &--home {
+            a {
+                border-bottom: $thinBorder #FFFFFF;  
+            }
+        }
+    }
+}
+
+header {
+    background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url(images/palmy.jpg), no-repeat;
+    text-align: center;
+    color: #FFFFFF;
+    display: grid;
+    @include grid (100%, repeat(2, 1fr));
+    align-items: center;
+}
+
+.header__title {
+    font-size: $titleFont;
+}
+
+.header__caption {
+    font-size: $biggerFont;
+    padding-bottom: $mainPadding;
+}
+
+.header__grey-area {
+    height: 200px;
+    border-right: $thickBorder $backgroundColor;
+    background-color: $darkBackgroundColor;
+    grid-column: {
+        start: 1;
+        end: span 1;
+    }
+    grid-row: {
+        start: 1;
+        end: 3;
+    }
+}    
+    
+.main-container {
+    padding: $mainPadding $mainPadding*2;
+}
+
+.about-me {
+    padding-bottom: $mainPadding;
+    display: grid;
+    grid-template-areas: 
```

Comment:
Ano, ale pozor na nadužívání `grid-template-areas`. :smile: Jak se vám otevřely pokročilejší koncepty, spousta lidí po nich sáhne spíš než třeba po flexboxu, ale někdy to dost na škodu. Jakmile si něco přímo pojmenuješ a přidala by sis další element, celá stránka by se ti rozbila a zvláštně rozházela. 

V tomhle případě se to asi měnit nebude, ale před každým dalším použitím `grid-template-areas` bych se zamyslela, jestli je to opravdu to nejjednoduší řešení, které případně bude jednoduché v budoucnosti změnit, pokud by ses rozhodla pro jiný design a podobně.